src/util/cmdline.h

```diff
@@ -112,6 +112,8 @@ class cmdlinet
     {}
   };
 
+  void parse_optstring(const char *optstring);
```

Comment:
:pick: Missing doxygen. Only a nitpick, as it was previously missing from the `parse` member function.